src/main/content/_layouts/post.html

```diff
@@ -55,7 +55,14 @@ <h1 id="post_title">{{ page.title | escape }}</h1>
                 <div class="language_container">
                     <span class="lang_text">{% t blog.post_in_other_lang %}:</span>
                     {% for i18n in page.blog-available-in-languages %}
-                        <a class="blog_lang" href="{{i18n.path}}">{% t langs.{{i18n.lang}} %}</a>
+                        {% if i18n.path contains 'https://openliberty.io' %}
+                            {% assign link_color = 'blue_link_light_background' %}
+                        {% elsif i18n.path contains 'https://' %}
+                            {% assign link_color = 'orange_link_light_background' %}
+                        {% else %}
+                            {% assign link_color = 'blue_link_light_background' %}
+                        {% endif %}
+                        <a class="blog_lang {{link_color}}" href="{{i18n.path}}">{% t langs.{{i18n.lang}} %}</a>
```

Comment:
This set of changes is for adding support for when a blog post in a different language is from a different domain. If it is an external domain, then style the link orange.

I opted not to use this code right now to keep things simple. The Simplified Chinese blog post itself already contains a link to the original English post.
<img width="1156" alt="image" src="https://user-images.githubusercontent.com/31117513/205750829-62c5a797-63e7-4cb0-8d1c-72d13569431e.png">